client/reducers/__tests__/leaderboard.test.js

```diff
@@ -0,0 +1,18 @@
+import { leaderboard_receiveData } from '@/actions'
+import leaderboardReducer from '@/reducers/leaderboard'
+import { petsWithScores } from '~/test/fake-data'
+
+describe('top 10 pets reducer', () => {
+  test('receives 10 top scoring pets', () => {
+    const action = {
+      type: leaderboard_receiveData,
+      leaderboard: petsWithScores,
+    }
+    const inputState = { data: [], error: null, loading: false }
+    const expectedOutputState = petsWithScores
+    const actualOutputState = leaderboardReducer(inputState, action)
+
+    expect(actualOutputState.leaderboard).toEqual(expectedOutputState)
+    expect(actualOutputState).not.toBe(inputState)
+  })
+})
```

Comment:
setError and requestData parts of reducer don't have a test

Reply:
Hi JV thanks for the feedback we have added test for these parts of the reducer. Thanks!